File: clients/java/src/main/java/org/anticaptrad/ActClient.java
package org.anticaptrad;
public final class ActClient { private final String baseUrl; public ActClient(String baseUrl){ this.baseUrl=baseUrl.replaceAll("/+$", ""); } public String url(String path){ return baseUrl + (path.startsWith("/") ? "" : "/") + path; } public String healthUrl(){ return url("/health"); } public String readyUrl(){ return url("/ready"); } }

File: clients/php/src/ActClient.php
<?php
namespace Anticaptrad;
final class ActClient { public function __construct(private string $baseUrl) { $this->baseUrl=rtrim($baseUrl,'/'); } public function url(string $path): string { return $this->baseUrl.'/'.ltrim($path,'/'); } public function healthUrl(): string { return $this->url('/health'); } public function readyUrl(): string { return $this->url('/ready'); } }
